packages/viz/src/lib/sources/Source.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * 
+ * Base Source definition. We should keep here the code shared between different sources 
+ */
+
+import { Credentials } from '@carto/toolkit-core';
+import { MapOptions } from '@carto/toolkit-maps';
+
+export interface sourceOptions {
+  credentials?: Credentials;
+  mapOptions?: MapOptions;
+}
+
+/* Interface definition of source blue print */
+export interface blueprint {
```

Comment:
There are some linter rules about naming, so blueprint so it should be `SourceOptions`, `Blueprint`... but we can review and fix them all once we have https://github.com/CartoDB/toolkit/pull/80 merged in